packages/main/src/components/top-navigation-bar/TopNavigationBar.jsx

```diff
@@ -265,7 +274,22 @@ export default function TopNavigationBar() {
               </NavLink>
             </li>
           </ul>
-
+          {isUserLoggedIn && (
+            <div className={TopNavigationBarStyles.notification}>
+              <NovuProvider
+                backendUrl={"http://139.144.17.179:3000"}
```

Comment:
why these IPs?

Reply:
The library allows you to integrate the backend this way

Reply:
No I mean why those IPs in particular

Reply:
That's the IP address for the server which the Backend hosted their endpoint on.

Reply:
so that's a personal hosting?

Reply:
if yes maybe you should talk to the BE mentors so they can setup in zuri core

Reply:
@ickynavigator it isn't hosted personally. 
The code for the server is in the messaging repo here https://github.com/zurichat/zc_messaging/tree/dev/notifications and was deployed by a devops. See [devops support thread](https://hng9.slack.com/archives/C04BE7MEJ5V/p1669318317926139)

The only reason it is an IP address here is that that IP hasn't been mapped to a zuri.chat domain. That's being worked on and should be up at notifications.zuri.chat eventually. In the meantime however, this would need to be merged to show the functionality has been implemented.

Reply:
@a11rew Alright, but this will have to be updated when it get's mapped.

Reply:
@a11rew oh alright boss
we can make a patch later on